testsuite/src/it/java/io/quarkus/ts/startstop/utils/MvnCmds.java

```diff
@@ -29,12 +31,22 @@
             new String[]{"mvn", "clean", "compile", "quarkus:build"},
             new String[]{"java", "-jar", "target/quarkus-runner.jar"}
     }),
-    //  DEV(new String[][]{
-    //          new String[]{"mvn", "clean", "compile", "quarkus:dev"}
-    //  }),
+    DEV(new String[][]{
+            new String[]{"mvn", "clean", "quarkus:dev"}
+    }),
     NATIVE(new String[][]{
             new String[]{"mvn", "clean", "compile", "quarkus:native-image", "-Pnative"},
```

Comment:
`quarkus:native-image` is no longer suggested way - see https://github.com/quarkusio/quarkus/blob/master/devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java#L41
The same for probably for`quarkus:build`

This TS should be using ways described in https://quarkus.io/guides/building-native-image
I would go with `mvn clean package -Pnative -DskipTests -DskipITs`

Similar adjustment should happen to `new String[]{"mvn", "clean", "compile", "quarkus:build"},` line

Reply:
Running `mvn clean verify` will probably hit some issues, maybe wotrh trying :-D
I filed few when playing with https://github.com/rsvoboda/rsvoboda-playground/blob/master/quarkus-extensions-combinations/foo.java